lua/lspconfig/server_configurations/gradle_ls.lua

```diff
@@ -20,6 +20,11 @@ return {
       return util.root_pattern(unpack(root_files))(fname) or util.root_pattern(unpack(fallback_root_files))(fname)
     end,
     cmd = { bin_name },
+    init_options = {
```

Comment:
maybe worth a 1-line comment here to avoid someone later removing this.

Reply:
I added a comment